src/oscar/apps/catalogue/abstract_models.py

```diff
@@ -128,6 +129,8 @@ class AbstractCategory(MP_Node):
 
     name = models.CharField(_('Name'), max_length=255, db_index=True)
     description = models.TextField(_('Description'), blank=True)
+    meta_title = models.CharField(_('Meta-title'), max_length=255, blank=True)
```

Comment:
Please change the label to "Meta title" instead of "Meta-title". Some for all other other fields below - please remove hyphen.